rts/at/util/src/main/java/org/jboss/jbossts/star/util/logging/at18NLogger.java

```diff
@@ -0,0 +1,124 @@
+package org.jboss.jbossts.star.util.logging;
+
+import static org.jboss.logging.Logger.Level.ERROR;
+import static org.jboss.logging.Logger.Level.WARN;
+
+import javax.transaction.xa.XAException;
+
+//import javax.json.JsonObject;
+
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * at18N log messages for the rest-at module.
+ */
+@MessageLogger(projectCode = "AT")
+public interface at18NLogger {
+	
+
+    /*
+        Message IDs are unique and non-recyclable.
+        Don't change the purpose of existing messages.
+          (tweak the message text or params for clarification if you like).
+        Allocate new messages by following instructions at the bottom of the file.
+     */
+	@LogMessage(level = ERROR)
+    @Message(id = 27001, value = "REST-AT transaction is required for this request. '%s'")
+    void error_rest_at_transactionrequired(int statusCode, @Cause Throwable t);
+
+    @LogMessage(level = ERROR)
+    @Message(id = 27002, value = "REST-AT transaction is not supported by this resource. '%s'")
+    void error_rest_at_transactionnotsupported(int statusCode,@Cause Throwable t);
+    
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27003, value = "Failed to start the bridge. '%s'")
+    void warn_rest_at_failedtostartbridge(String response, @Cause Throwable t);
+	
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27004, value = "Failed to stop the bridge. '%s'")
+    void warn_rest_at_failedtostopbridge(String response, @Cause Throwable t);
+    
+    
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27005, value = "Failed to enlist inbound bridge to the transaction '%s'")
+    void warn_rest_at_failedtoenlisttransaction(String response, @Cause Throwable t);
+    
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27006, value = "Failed to import transaction. '%s'")
+    void warn_rest_at_failedtoimporttransaction(String response, @Cause Throwable t);
+    
+    @LogMessage(level = ERROR)
+    @Message(id = 27007, value = "FATAL System Exception '%s'")
+    void error_rest_at_systemexception(String cause);
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27008, value = "Object Store Exception. '%s'")
+    void warn_rest_at_objectStoreException(String cause, @Cause Throwable t);
+    
+    
+    @LogMessage(level = WARN)
+    @Message(id = 27009, value = "IOException. '%s'")
+    void warn_rest_at_IOException(String cause, @Cause Throwable t);
```

Comment:
I understand that this message was probably taken from the message used before in the code. But message which says "IOException" and then puts "IOException" to the details of the log (caused by `@Cause` annotation) has no much sense. It would be good if the message (and the name of the method) could be change to say like "cannot read suoradinate atomic action from recovery store" or something similar. Plus, using the `e.getMessage()` as message when the log contains whole stacktrace (again the `@Cause` is used here) is ambiguous - the `e.getMessage` will be printed twice without no good reason.

The same points are true for `warn_rest_at_objectStoreException` method.